### DISCUSSION TITLE
Trend of lattice thermal conductivity with increasing k-points and SCP+BTE approach

### DISCUSSION THREAD

**Discussion by AbhiyanPandit** [ttadano/alamode#56] [Q&A]
Dear Dr. Tadano,

Based on the ALAMODE implementation, I want your suggestions on the following cases:
1) Should the lattice thermal conductivity (LTC) always keep increasing with the increased k-points until the convergence? For my system the LTC with 200*200*1 k-points is lower than that with the 180*180*1 k-points, but it increases with  the 220*220*1 k-points. If this trend is not normal, then what factor(s)/parameters should be checked?
2) Is the LTC with SCP+BTE approach always expected to increase than that obtained using the conventional BTE approach? I find the LTC increases with the SCP+BTE approach in some articles (https://doi.org/10.7566/JPSJ.87.041015, 10.1103/PhysRevLett.120.105901).

Best,
Abhiyan

**Comment by ttadano** [comment]
1. It is possible that the LTC value decreases when you increase the k-mesh density slightly, but as you observed, a denser k-mesh tends to give a larger LTC. 
2. No. If the SCP frequency is lower than the harmonic frequency, I think the LTC value of the SCP+BTE would be lower than that of HA+BTE.

**Reply by AbhiyanPandit** [reply]
Dear Dr. Tadano,

Thank you very much for the informative response.

Best,
Abhiyan
